apps/docs-snippets/test/fixtures/abi/encode-and-decode.jsonc

```diff
@@ -1,30 +1,25 @@
 // #region encode-and-decode-2
 {
-  "encoding": "1",
-  "types": [
+  "programType": "script",
```

Comment:
Just noticed we have `programType` in the ABI now cc #989

Reply:
Updated #989 and removed blocked label.

Reply:
Technically, that is still blocked until the new `forc` version comes out, but that should happen soon. 🙂